### PR TITLE
[CPDP-1602] - For security we deleted a document - mandatory evidence

### PR DESCRIPTION
**Pull request by modivilioglu**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -2435,11 +2435,10 @@ mandatoryEvidence.helpText=Files can be up to 3MB. We accept these file types XL
 
 mandatoryEvidence.cyaChange=the document showing how the Capital Gains Tax due was worked out
 
-mandatoryEvidenceExpired.title=Your uploaded file has expired
-mandatoryEvidenceExpired.p1=Uploaded files can only be kept for 5 days before they expire. You must either upload the file again or replace it with another before you can continue.
-mandatoryEvidenceExpired.label=Supporting file
-mandatoryEvidenceExpired.cyaChange=the file or upload it again
-
+mandatoryEvidenceExpired.title=For your security, we deleted this document
+mandatoryEvidenceExpired.p1=We deleted the document because we only save documents for 5 days. <br>You’ll need to try again and upload a document showing how the amount of Capital Gains Tax due was worked out.
```
Comment (AndyHWChung) [inline conf/messages]: doesn't it work if you just have another `<p>` instead of a `<br>`?

Comment (modivilioglu) [reply conf/messages]: I've changed it already <br> surprisingly didnt work :/

File: conf/application.conf [hunk 2]
```diff
@@ -108,7 +108,7 @@ microservice {
         port = 9570
         max-uploads = 2
         max-file-size = 3145728
-        s3-url-expiry-duration = 5days
+        s3-url-expiry-duration = 5seconds
```
Comment (AndyHWChung) [inline conf/application.conf:111]: this should be changed back to the default 5days

Comment (modivilioglu) [reply conf/application.conf]: Done. Very sorry.

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**